client_internals_it_test.go

```diff
@@ -421,6 +423,32 @@ func clientInternalEncodeDataTest(t *testing.T) {
 	})
 }
 
+func clientClusterConnectionConfigRetryTimeTest(t *testing.T) {
+	// TODO: Adapt this test for t.Parallel()
+	//t.Parallel()
+	ctx := context.Background()
+	const ASSERTION_SECONDS = 30
+	port := it.NextPort()
+	cls := it.StartNewClusterWithOptions("dev", port, 1)
+	defer cls.Shutdown()
+	config := cls.DefaultConfig()
+	config.Cluster.ConnectionStrategy.Retry.InitialBackoff = math.MaxInt32
+	config.Cluster.ConnectionStrategy.Retry.MaxBackoff = math.MaxInt32
+	client := it.MustClient(hz.StartNewClientWithConfig(ctx, config))
+	ci := hz.NewClientInternal(client)
+	ret, err := cls.RC.TerminateMember(ctx, cls.ClusterID, cls.MemberUUIDs[0])
+	require.NoError(t, err)
+	require.True(t, ret)
+	cm := ci.ConnectionManager()
+	require.True(t, len(cm.ActiveConnections()) == 0)
```

Comment:
Could you change this line to the following to make it similar to the reference implementation test:
```
it.Eventually(func() bool {
    return len(cm.ActiveConnections()) == 0
})
```

Reply:
Fixed at https://github.com/hazelcast/hazelcast-go-client/pull/861/commits/830865005e9c8f61b0599f85a7fb25de30d862fc